utils/config/daos_server.yml

```diff
@@ -70,12 +70,13 @@
 #
 ## Network provider
 #
-## Force a specific libfabric provider to be used by all the engines.
-## The default provider depends on the interfaces that will be auto-detected:
-##  ofi+verbs for Infiniband/RoCE and
-##  ofi+tcp for non-RDMA-capable Ethernet.
+## Set the libfabric provider to be used by all the engines.
```

Comment:
I would replace libfabric with just "fabric" since we are adding UCX now.

Reply:
done